### ISSUE TITLE
New WatchFace to give description of current time accurate to a minute

### ISSUE THREAD

**Issue by bmsleight** [InfiniTimeOrg/InfiniTime#1305]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

New Words WatchFace - PR to follow

### Description

PR to follow see screen shots

**Comment by bmsleight** [comment]
![InfiniSim_2022-08-28_132955](https://user-images.githubusercontent.com/3758389/187076611-9e23143e-3d3b-48f6-be7a-d463eb763e50.png)
![InfiniSim_2022-08-28_131613](https://user-images.githubusercontent.com/3758389/187076612-7f707f5c-48a9-4234-ba72-ea46b6e3868a.png)
![InfiniSim_2022-08-28_131438](https://user-images.githubusercontent.com/3758389/187076613-4b2877f1-f986-4102-ad31-8d88b2ada749.png)

**Comment by hrmckay** [comment]
There is a similar watch face called Wordclock on this site:  https://zephyrlabs.github.io/Watchfaces/

**Comment by bmsleight** [comment]
Yes similar, but my preference  "Almost quarter past one in the afternoon"  to "fourteen past one" [13:14] and I prefer "Just gone twenty to nine in the evening" to "nineteen minutes to nine" [20:41]

**Comment by bmsleight** [comment]
Dates in words as well
![InfiniSim_2022-08-28_200900](https://user-images.githubusercontent.com/3758389/187092649-a0bb1b0a-bb5d-4c86-a6d9-40ae2985c82e.png)

**Comment by kieranc** [comment]
There's also this PR: #660

**Comment by Riksu9000** [comment]
Not sure what the point of this issue is. We don't have a convention of having an issue for each PR.
